test/index.js

```diff
@@ -148,7 +164,23 @@ describe('expect()', function () {
         done();
     });
 
-    it('asserts on invalid condition (long object values)', function (done) {
+    it('asserts on invalid condition (large object, display not truncated)', function (done) {
+
+        var exception = false;
+        try {
+            Code.truncate = false;
+            Code.expect({ a: 1, b: 2, c: 3, d: 4, e: 5, f: 6, g: 7, h: 8, i: 9, j: 10 }).to.be.a.string();
+        }
+        catch (err) {
+            Code.truncate = true;
+            exception = err;
+        }
+
+        Hoek.assert(exception.message === 'Expected { a: 1, b: 2, c: 3, d: 4, e: 5, f: 6, g: 7, h: 8, i: 9, j: 10 } to be a string but got \'object\'', exception);
```

Comment:
Minor nit: `exception.message` will throw an error if `exception` is still equal to `false`. I don't think you need to change it though, as the test should be failing and that style is used throughout the file.